WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -1113,6 +1113,11 @@ public void updatePostContent(boolean isAutoSave) {
      */
 
     private void fetchMedia(Uri mediaUri) {
+        if (mediaUri == null) {
```

Comment:
This `fetchMedia` method is called when the user selects one video or one picture from the local library. So maximum one toast at a time when the user comes back the image/video picker.